src/mui/_compact.scss

```diff
@@ -0,0 +1,21 @@
+// @see https://github.com/Igosuki/compass-mixins/issues/34
```

Comment:
It was as we used some `transform` mixins, which are broken without this file. Not used anymore.